docs/ug1.md

```diff
@@ -0,0 +1,326 @@
+    * **`addCompany`**`n/Google p/98765432 e/hr_google@gmail.com a/70 Pasir Panjang Rd, #03-71 Mapletree Business City II, Singapore 117371` :
+      Adds a company named `Google` to Tinner.
+    * **`addRole`**`1 n/Software Engineering Intern s/applying r/31-03-2022 23:59 d/Write mobile applications $/5000` :
+      Adds a software engineering intern role to the company at index 1.
+    * **`deleteCompany`**`1` : Deletes the 1st company shown in the current list.
+    * **`deleteRole`**`1 1` : Deletes the 1st role of the 1st company shown in the current list.
+    * **`exit`** : Exits the app.
+
+6. Refer to the [Features](#features) below for details of each command.
+--------------------------------------------------------------------------------------------------------------------
+
+## Demo Use Cases <a id="demos"></a>
+
+### Use Case #1: Applying to a role in a company
+
+Say you found a company with an opening that you are interested in. You did your research and found relevant information pertaining to the company and the role. Here's how you can input this data in the application:
+
+1. [Add the company](#c-add-c) to the company list.
+   ![AddCompanyDemo](images/AddCompanyDemo.png)
+
+
+2. [Add the role](#c-add-c-r) under the added company.
+   ![AddRoleDemo](images/AddRoleDemo.png)
+
+
+
+
+--------------------------------------------------------------------------------------------------------------------
+## Features <a id="features"></a>
+
+<div markdown="block" class="alert alert-info">
+
+**:information_source: Notes about the command format:**<br>
+
+* Words in `UPPER_CASE` are the parameters to be supplied by the user.<br>
+  e.g. in `add n/COMPANY`, `COMPANY` is a parameter* which can be used as `add n/Google`.
+
+* Items in square brackets are optional.<br>
+  e.g `n/COMPANY [p/PHONE_NUMBER]` can be used as `n/Google P/65427981` or as `n/Google`.
+
+* Parameters can be in any order.<br>
+  e.g. if the command specifies `n/COMPANY [p/PHONE_NUMBER]`, `[p/PHONE_NUMBER] n/COMPANY` is also acceptable.
+
+* If a parameter is expected only once in the command but you specified it multiple times, only the last occurrence of the parameter will be taken.<br>
+  e.g. if you specify `p/12341234 p/56785678`, only `p/56785678` will be taken.
+
+* Extraneous parameters for commands that do not take in parameters (such as `help`, `list`, `exit`) will be ignored.<br>
+  e.g. if the command specifies `help 123`, it will be interpreted as `help`.
+
+*Note: a parameter is the user input that is given to a command.
+</div>
+
+### Listing all companies : `list` <a id="c-list"></a>
+
+Shows a list of all companies and internship roles in Tinner.
+
+Format: `list`
+
+### Listing all favourited companies : `listFavourite` <a id="c-listfavourite"></a>
+
+Shows a list of all favourited companies and internship roles within these companies in Tinner.
+
+Format: `listFavourite`
+
+### Adding a company: `addCompany` <a id="c-add-c"></a>
```

Comment:
Perhaps here we can also link them to the individual use case demos?

Reply:
Will do 👍